docs/examples/algo/complete-example.py

```diff
@@ -1,87 +1,106 @@
 import csv
 import datetime
-
 import numpy
+import os
+import yaml
```

Comment:
Python model params adds the extra requirement of a local import, an extra directory, and an empty `__init__.py` file.